openfisca_us/tests/policy/baseline/usda/snap/integration.yaml

```diff
@@ -13,18 +14,18 @@
         age: 15
     spm_units:
       spm_unit:
-        snap_gross_income: 2000 * 12
         members: [person_1, person_2, person_3, person_4]
+        is_snap_eligible: true
```

Comment:
Without this, `snap` was getting set to zero. The $2,000/mo income is below the gross income limit of $2,839 (130% of FPL for a family of four) and the net income limit of $2,184 (100% of FPL) so I haven't figured out why this was necessary.

Reply:
Just took a look - @nmrodelo I think this was because the net/gross income limit percentages were getting multiplied by 12, rather than by the FPL.